en/includes/guides/customize/actions/understanding-actions.md

```diff
@@ -182,7 +182,15 @@ Requests will not be retried if the external service responds with HTTP status c
 
 ### Troubleshooting
 
-!!! note
-    Troubleshooting logs are not yet incorporated but are planned for inclusion by end October 2024.
+You can use diagnostic logs to capture requests sent from the identity server, responses received from your external service, the status and context data for response processing, helping you monitor and troubleshoot issues efficiently . Each event entry provides key details like timestamps, result status, result message, error codes and other relevant context.
```

Comment:
Feels like the very first sentence is bit longer and hard to understand. Shall we split it.

Reply:
Updated in https://github.com/wso2/docs-is/pull/4819/commits/a877737fe142a2f9c37e955b8dd2de7749da8f57